### PR TITLE
accumulate: add canonical data

### PR DESCRIPTION
**Pull request by SleeplessByte**

### CODE REVIEW

**Review by SaschaMann — APPROVED** [review]

File: exercises/accumulate/canonical-data.json [hunk 1]
```diff
@@ -0,0 +1,113 @@
+{
+  "exercise": "accumulate",
+  "description": [
```
Comment (SaschaMann) [inline exercises/accumulate/canonical-data.json:3]:
```suggestion
  "comments": [
```

**Comment by SleeplessByte** [comment]
Thanks! That was a tiny mistake 😅

File: exercises/accumulate/canonical-data.json [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  "comments": [
+    " JSON doesn't allow for functions, and this exercise ",
+    " passes in a function as one of the inputs. These    ",
+    " inputs are represented using pseudocode (string).   ",
+    " It's up to the test generator to turn that string   ",
+    " into a function declaration.                        "
+  ],
+  "cases": [
+    {
+      "description": "accumulate",
```
Comment (ErikSchierboom) [inline exercises/accumulate/canonical-data.json:12]: Why this extra layer of indirection?

**Review by ErikSchierboom — APPROVED** [review]
Great! I've left one comment.

File: exercises/accumulate/canonical-data.json (hunk 2 above)
Comment (SleeplessByte) [reply exercises/accumulate/canonical-data.json:12]: Consistency with other files. I don't care. We can remove :D

Comment (ErikSchierboom) [reply exercises/accumulate/canonical-data.json:12]: I'd prefer removal. I think we once did a pass to try and remove the unneeded levels of indirection, but we probably didn't fully see that through

Comment (SleeplessByte) [reply exercises/accumulate/canonical-data.json:12]: Got it!; Ill edit.

**Comment by SleeplessByte** [comment]
For the person merging: Squash!

**Review by valentin-p — APPROVED** [review]

**Pull request merged**